CHANGELOG.md

```diff
@@ -7,14 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+The module was **rewritten to TypeScript.** It should be fully backwards compatible, but if you run into any issues, [please report them](https://github.com/superfaceai/passport-twitter-oauth2/issues).
```

Comment:
```suggestion
### Changed
- The module was **rewritten to TypeScript.** It should be fully backwards compatible, but if you run into any issues, [please report them](https://github.com/superfaceai/passport-twitter-oauth2/issues).
```

I am afraid this is not valid CHANGELOG format.

Reply:
There is already `Changed` section, the suggestion can not be used.

Reply:
Right. I think it is fine from the convention's point of view, but dunno if parser will accept it. So I will move the Changed section to top and add this as a first point.

Reply:
We should add validation of changelog format to the testing pipeline.